devel/ci/cico.pipeline

```diff
@@ -45,6 +45,9 @@ def configure_node = {
     onmyduffynode 'yum -y install epel-release'
     onmyduffynode 'yum install -y docker python36-click rsync'
     onmyduffynode 'systemctl start docker'
+    // To run the integration testsuite
+    onmyduffynode 'python3.6 -m ensurepip'
+    onmyduffynode 'python3.6 -m pip install \'pytest<4.0\' pytest-cov conu'
```

Comment:
Do the tests not work with ```pytest 4```? Also, we could run the test suite itself inside a container so we don't need to package all the test suite dependencies for EPEL.

Reply:
Ah yeah I guess that could be a problem, though I am not sure. Hmm. I guess we can just go this direction for now.

Reply:
I tried with Pytest 4 again, it fails because pytest-cov tries to use a deprecated API (`config.warn`) and there is no pytest-cov update available.